src/filer.js

```diff
@@ -564,13 +569,11 @@ var Filer = new function() {
           }
         },
         function(e) {
-          if (e.code == FileError.INVALID_MODIFICATION_ERR) {
-            e.message = "'" + path + "' already exists";
-            if (opt_errorHandler) {
-              opt_errorHandler(e);
-            } else {
-              throw e;
-            }
+          e.message = e.message || "'" + path + "' already exists";
```

Comment:
I think we still want the check for `FileError.INVALID_MODIFICATION_ERR`. IIRC, the `e.message` is not informative. Hence the custom one.